plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -1144,10 +1144,10 @@ actions:
         description: String of the XQL query
         type: string
         required: true
-        example: dataset=xdr_data | fields event_id, event_type, event_sub_type | limit 3
+        example: dataset=xdr_data I fields event_id, event_type, event_sub_type I limit 3
```

Comment:
Does the letter I work as well as the | character for queries?

Reply:
Nope. Actually, it just seems to be looking the same way as `|` character. Putting that in quotes did change nothing. It was breaking the markdown table as well 😞